main.roc

```diff
@@ -1,7 +1,7 @@
 app "generate-build"
-    packages { pf: "/Users/luke/Documents/GitHub/roc/examples/static-site-gen/platform/main.roc" }
+    packages { pf: "/home/runner/work/examples/examples/roc_nightly/examples/static-site-gen/platform/main.roc" }
```

Comment:
this is the path used by the CI machine

Reply:
Ok, we should update README specifically in `## To view the examples site locally` section it still says to use `roc run main.roc -- examples build` which doesn't work with this change.